test/services/PayApiServiceSpec.scala

```diff
@@ -445,8 +447,8 @@ class PayApiServiceSpec extends BaseSpec {
       val uiWithBstArrival: UserInformation = userInformation.copy(
         selectPlaceOfArrival = "",
         enterPlaceOfArrival = "LHR",
-        dateOfArrival = LocalDate.parse("2018-7-12"),
-        timeOfArrival = LocalTime.parse("12:20 pm", DateTimeFormat.forPattern("hh:mm aa"))
+        dateOfArrival = LocalDate.parse("2018-07-12"),
```

Comment:
this should work as 7 if I understand pattern corrctly please revert from 07 to 7 and
use your new method parseLocalDate